Library/src/Multiplayer/MultiplayerConnection.cpp

```diff
@@ -627,6 +634,46 @@ void MultiplayerConnection::ResetScopes(ErrorCodeCallbackHandler Callback)
 	Connection->Invoke("ResetScopes", Params, LocalCallback);
 }
 
+void MultiplayerConnection::LeaveScopes(ErrorCodeCallbackHandler Callback)
```

Comment:
created this on the same model as SetScopes and ResetScopes when I realised that ExitSpace was not firing the LeaveScopes event (is this needed???)